vue/components/organisms/ripe-configurator/ripe-configurator.vue

```diff
@@ -313,6 +313,7 @@ export const RipeConfigurator = {
         },
         configProps: {
             handler: async function(value) {
+                this.partsData = this.parts;

```

Comment:
Why isn't this updated by the `parts` watcher?

Reply:
It is updated by the `parts` watcher, however, when a new brand/model is given, this runs first.  When `sdk's .config()` is called, the `partsData` has the parts of the old model, resulting in errors by the `sdk`.

Reply:
Can you find out if it's possible to have control over the order by which the watchers fire? It's not a great practice and we'll end up with this fix anyway, is just so that we understand how is that defined (if it's deterministic).

Reply:
So, I was wrong. The flow of events is this:
1. When the model/brand changes, the watcher on the `parts` runs first and the watcher on the `configProps` is next. This order changes it the watcher for the `configProps` is moved to be positioned above the `parts` watcher.
2. By leaving the order as it was, the error in the console happens because the `setPartsRipe` was being called before the `configRipe`, with the old model defaultConfig but the correct parts for the new model https://github.com/ripe-tech/ripe-sdk/blob/f2f47516129cfe937388bef19c9baa4deeb6c5ea/src/js/base/ripe.js#L1515. This resulted in an error by the sdk but eventually the configuration would be made correct.

To fix this and leave the order as is, a check must be made in the `parts` watcher, and not calling the `setPartsRipe` if the brand/model/version `props` differ from the ones in the current config of the sdk, since the call to change (including the setting of new parts) will be made by the `configRipe` that is called next.

Reply:
Maybe we should be changing the order then. WDYT?

Reply:
I tried it, the problem remains. The `setPartsRipe` happens in the middle of the `setConfig`, resulting in using the outdated configuration to set the parts, which is later updated. The change is applied all the same, but the error still happens and appears in the console.